dist/XSLoader/XSLoader_pm.PL

```diff
@@ -11,7 +11,6 @@ print OUT <<'EOT';
 
 package XSLoader;
 
-use p5;
```

Comment:
This is globally the right thing to do.
I'm just concerned that this is going to break XSLoader and DynaLoader.... 
and if so we should fix them at the same time?